spec/granite_orm_spec.cr

```diff
@@ -105,4 +109,26 @@ describe Granite::ORM::Base do
       s.to_json.should eq %({"custom_id":3,"name":"Hacker News"})
     end
   end
+
+  describe "validate fields" do
```

Comment:
There are some rules, taken from http://www.betterspecs.org/, that could be applied here to improve the readability of the test. I'd write the following.

```
describe "validating fields"
  context "without a name" do
    it "is not valid" do
      s = Website.new(name: "")

      s.valid?.should eq false
      s.errors.should contain "Name cannot be blank"
    end 
  end

  context "when name is present" do
    it "is valid" do
      s = WebSite.new(name: "Hacker News")
      
       s.valid?.should eq true
       s.errors.empty?.should eq true
    end
  end
end